addons/udes_stock/models/stock_move_line.py

```diff
@@ -1038,6 +1038,25 @@ def _prevent_write_after_done(self):
         if done_lines:
             raise ValidationError(_("Cannot update move lines that are already 'done'."))
 
+    @api.model_cr
+    def init(self):
+        """ Creates indexes for _check_resultant_package_level
+        """
+        super(StockMoveLine, self).init()
+
+        tools.create_index(
+            self._cr,
+            "stock_move_line_state_result_package_id_index",
+            self._table,
+            ["state", "result_package_id"],
```

Comment:
If we're theoretically targeting Python3.5, I think the trialing comma on the final arguemnt isn't valid syntax.

Reply:
Here I was following the existing style for this file (eg. line 822)